[DealabsBridge - HotUKDealsBridge - MydealsBridge] Update Groups

sysadminstory opened the pull request:
The bridges has been updated with the newest "groups" available on every
website !

sysadminstory committed:
[DealabsBridge - HotUKDealsBridge - MydealsBridge] Update Groups

The bridges has been updated with the newest "groups" available on every
website !

em92 commented on bridges/DealabsBridge.php:
```diff
@@ -382,21 +750,42 @@ class DealabsBridge extends PepperBridgeAbstract {
 					'Hubs' => 'hubs',
+					'Hugo Boss Bottled' => 'hugo-boss-bottled',
 					'Huile moteur' => 'huile-moteur',
-					'Hygiène corporelle' => 'hygiene-corporelle',
```
Links with `hygiene-corporelle` become invalid.

em92 commented on bridges/DealabsBridge.php:
```diff
@@ -382,21 +750,42 @@ class DealabsBridge extends PepperBridgeAbstract {
+					'Hygiène &amp; soins' => 'hygiene-soins',
 					'Hygiène de la maison' => 'hygiene-de-la-maison',
 					'Hygiène des bébés' => 'hygiene-des-bebes',
-					'Image, son & vidéo' => 'image-son-video',
```
Links with `image-son-video` become invalid.

em92 commented on bridges/DealabsBridge.php:
```diff
@@ -482,161 +963,317 @@ class DealabsBridge extends PepperBridgeAbstract {
 					'Magazines' => 'magazines',
 					'Maillots de bain' => 'maillots-de-bain',
 					'Maillots de football' => 'maillots-de-football',
-					'Maison & Jardin' => 'maison-et-jardin',
```
Same

em92 commented on bridges/DealabsBridge.php:
```diff
@@ -482,161 +963,317 @@ class DealabsBridge extends PepperBridgeAbstract {
+					'Micro SD' => 'micro-sd',
+					'Microsoft Flight Simulator' => 'microsoft-flight-simulator',
 					'Microsoft Office' => 'microsoft-office',
-					'Microsoft Surface' => 'microsoft-surface',
```
Same. Didn't check other values that were removed

em92 reviewed (changes requested):
Started checking, if old feed links are still valid on those values, that were removed. Found some, that became invalid.

sysadminstory commented:
Some of the existing links (like 'image-son-video') are redirected to another URL on the website, and some (like 'microsoft-surface') don't exists anymore.
But some (like 'maison-et-jardin') are still existing, even I think I crawled the whole website using a script...

I'll check if some of them are still visible in the "Similar Groups" box.
I must check my crawling script again !

sysadminstory committed:
[DealabsBridge - HotUKDealsBridge - MydealsBridge] Update Groups : double check

Groups were extracted from :
- Website menu and the links to groups categories
- Group categories Popular Groups and "More similar groups" links
- Group page breadcrumb
- Group page similar grous

sysadminstory commented:
Hello !

I double checked : I extended my crawling script, to get more "sources" : some of the existing groups (like 'maison-et-jardin') does still work, but there are not on the website anymore. I think those are abandoned groups : the last deal in this group is from 2019.

The group order changed a bit, because I fixed a duplicate entries issue in my crawling script, and did not need to use the sort command to fix them.

IMHO, the feed this pull request will break are already broken, or does not return anything new since a while.

sysadminstory commented:
Did I miss something ? :)

sysadminstory commented:
Hello !

Am I missing some changes ?

em92 commented:
gj!

em92 commented:
Btw, just pushed https://github.com/RSS-Bridge/rss-bridge/commit/877707f7b2bf283298694687954773fe6503438a, so you can add your script crawling scripts to `contrib/` directory
